### PR TITLE
Linux Rules Update

### PR DESCRIPTION
**Pull request by nasbench**
Hi,
- This PR has some updates to the Linux rules
  - Unifying indentation
  - Add new flags to older rules where it make sense
  - New rules covering curl, service stop and other.
- A small update to the windows curl rule has also been pushed

### CODE REVIEW

**Comment by nasbench** [comment]
cc: @frack113 @Neo23x0

File: rules/windows/process_creation/proc_creation_win_susp_curl_fileupload.yml [hunk 1]
```diff
@@ -21,10 +21,12 @@ detection:
         CommandLine|contains:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_curl_fileupload.yml:21]:
```yaml
        CommandLine|contains:
            - ' -F '
            - ' --form' # get the  --form-string too
            - ' -T '
            - ' --upload-file '
            - ' -d '
            - ' --data' # For flags like: "--data ", "--data-ascii", "--data-binary", "--data-raw", "--data-urlencode"
```

File: rules/linux/process_creation/proc_creation_lnx_local_groups.yml [hunk 2]
```diff
@@ -6,15 +6,18 @@ author: Ömer Günal, Alejandro Ortuno, oscd.community
...
+modified: 2022/09/15
 logsource:
     category: process_creation
     product: linux
 detection:
     selection_1:
         Image|endswith: '/groups'
     selection_2:
-        Image|endswith: '/cat'
+        Image|endswith:
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/linux/process_creation/proc_creation_lnx_local_groups.yml:17]: Yes. I will add it

File: rules/linux/process_creation/proc_creation_lnx_services_stop_and_disable.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+title: Disable Or Stop Servicess
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_services_stop_and_disable.yml:1]: "Servicess" so many ?

**Review by frack113 — APPROVED** [review]

**Pull request merged**